packages/web-app-files/src/components/SideBar/Shares/RoleDropdown.vue

```diff
@@ -158,7 +170,14 @@ export default {
 
   methods: {
     applyRoleAndPermissions() {
-      this.selectedRole = this.existingRole || PeopleShareRoles.list(this.resource.isFolder)[0]
+      if (this.existingRole) {
```

Comment:
I'd prefer a syntax with an early return, i.e. like the following: 
```js
if (this.existingRole) {
  this.selectedRole = this.existingRole
  return
}

this.selectedRole = this.resourceIsSpace ? SpacePeopleShareRoles.list()[0] : PeopleShareRoles.list(this.resource.isFolder)[0]
```

Reply:
But the method does more than just that. There is a second if-block, which I think should not be omitted by an early return.